src/chart.zoom.js

```diff
@@ -343,6 +357,7 @@ var zoomPlugin = {
 			mc.on('panend', function(e) {
 				currentDeltaX = null;
 				currentDeltaY = null;
+                zoomNS.panCumulativeDelta = 0;
```

Comment:
Not sure why. My copy looks lined up. Have deleted all indents and reset it with tabs